### DISCUSSION TITLE
Flag avx not available for AMD processor on some virt machines.

### DISCUSSION THREAD

**Discussion by lvs1234** [cockpit-project/cockpit#18300] [Q&A]
Hello!
Make 5-10 VM on Dedicated Server with AMD processor with avx. Some of VM has avx flag, some doesn't have. How I can make enable flag avx on all VM? 
Try to find in google. Solution only for Virtual Box. Change KVM to host in virtual box dashbord.

Owner of dedicated server check BIOS, manually switch off power and turn on. 
Nothing help.

**Comment by marusak** [comment]
Have you used Machines cockpit plugin for creating of these VMs?

**Reply by lvs1234** [reply]
Yes.
